### DISCUSSION TITLE
Owncast 0.1.1 amd64-binary from last month is from apr 2020

### DISCUSSION THREAD

**Discussion by Epy** [owncast/owncast#3281] [Q&A]
Hello
I didn't make an issue because I'm not sure of what's happening, it could be nothing as I don't know much about Go applications.

I downloaded the latest release on this page: https://github.com/owncast/owncast/releases v0.1.1, amd64 version.
When I unzippped the file on the server, the binary says it's from April 2020

Tried on my computer, it's the same date: 
![Owncast zipped](https://github.com/owncast/owncast/assets/450856/c96a8f11-074d-43c3-a995-71b40ddb917f)

Is that normal ? I would have thought the date would be the date of the last compilation of the binary, so "July 28th" of this year

Thanks

**Comment by gabek** [comment]
Filed as #3282.
